solutions/usaco-764.mdx

```diff
@@ -146,4 +154,44 @@ public class shuffle {
 
 </JavaSection>
 
+<PySection>
+
+```python
+from collections import deque
+
+with open("shuffle.in", "r") as input_file:
```

Comment:
i'd use single quotes for python, don't really care about this though

Reply:
oop too used to c++